kafka/src/main/java/com/bazaarvoice/emodb/kafka/KafkaModule.java

```diff
@@ -24,9 +28,21 @@ String provideBootstrapServers(KafkaConfiguration kafkaConfiguration) {
 
     @Provides
     @Singleton
-    AdminClient provideAdminClient(@BootstrapServers String bootstrapServers) {
+    AdminClient provideAdminClient(@BootstrapServers String bootstrapServers, @Nullable SslConfiguration sslConfiguration) {
         Properties properties = new Properties();
         properties.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers);
+
+        if (null != sslConfiguration) {
```

Comment:
If you want to leave this I won't object, but we Emo convention has been to put the variable first.

Reply:
the reason i put the literal on the left is that you won't overwrite it if accidentally use `=` instead of `==`